hudi-cli/src/main/java/org/apache/hudi/cli/commands/TableCommand.java

```diff
@@ -106,10 +106,13 @@ public String createTable(
       throw new IllegalStateException("Table already existing in path : " + path);
     }
 
-    final HoodieTableType tableType = HoodieTableType.valueOf(tableTypeStr);
-    HoodieTableMetaClient.initTableType(HoodieCLI.conf, path, tableType, name, archiveFolder,
-        payloadClass, layoutVersion);
-
+    new HoodieTableConfig.PropertyBuilder()
```

Comment:
WDYT about adding a static method named e.g. `HoodieTableConfig.propertyBuilder()` so that we can make the code more readable?

Reply:
Sound good! I will have a try.

Reply:
Hi @yanghua , I have update the PR. Please take a look again when you have time. Thanks~